h2/stream.py

```diff
@@ -279,19 +315,40 @@ def send_reset(self, previous_state):
         it's the first time we've done this if the stream is currently in a
         state other than CLOSED.
         """
-        events = []
+        assert previous_state != StreamState.CLOSED
 
-        if previous_state != StreamState.CLOSED:
-            event = StreamReset()
-            event.stream_id = self.stream_id
-            event.error_code = ErrorCodes.STREAM_CLOSED
-            event.remote_reset = False
-            events.append(event)
+        self.stream_closed_by = StreamClosedBy.SEND_RST_STREAM
 
         error = StreamClosedError(self.stream_id)
-        error._events = events
+
+        event = StreamReset()
+        event.stream_id = self.stream_id
+        event.error_code = ErrorCodes.STREAM_CLOSED
+        event.remote_reset = False
+        error._events = [event]
```

Comment:
Because in current version, this method never be called at CLOSE state.
So the "if condition" never be false and I remove the "if condition" for the coverage.
Should I add a comment on this?